src/action_processer_utils.rs

```diff
@@ -313,6 +315,21 @@ fn configure_edit_scale_uniform(action_processer_builder: &mut ActionProcesserBu
         );
 }
 
+pub fn configure_command_input(action_processer_builder: &mut ActionProcesserBuilder){
```

Comment:
Does this need to be public?

Reply:
No, it doesn't. will change it to private